src/scanpy/readwrite.py

```diff
@@ -579,6 +580,12 @@ def read_10x_mtx(
         if the files are named `patientA_matrix.mtx`, `patientA_genes.tsv` and
         `patientA_barcodes.tsv` the prefix is `patientA_`.
         (Default: no prefix)
+    compressed
+        Whether to expect Cell Ranger v3+ files (.mtx, features.tsv, barcodes.tsv)
+        to be gzipped. If True, '.gz' suffix is appended to filenames.
+        Set to False for STARsolo output.
+        Has no effect on legacy (v2-) files.
+        (Default: True)
```

Comment:
```suggestion
```

The default is already defined in the docstring so we don't need it here anymore.